functional-tests/test-bed/src/test/java/org/arquillian/smart/testing/ftest/testbed/ProjectPersistFail.java

```diff
@@ -0,0 +1,13 @@
+package org.arquillian.smart.testing.ftest.testbed;
+
+import org.arquillian.smart.testing.ftest.TestBedTemplate;
+import org.junit.Assert;
+import org.junit.Test;
+
+public class ProjectPersistFail extends TestBedTemplate {
+
+    @Test
+    public void should_fail() throws Exception {
+        Assert.assertFalse(true);
```

Comment:
How about making it an inner class for `ProjectPersistTest` - it would be easier to understand the purpose of this test class.